src/shared/common/components/workstack-tag.jsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+const { fetchTagType } = require('../../../../server/config/tagType/tagType');
 
 export default function Tags(props) {
     let tags = [];
 
     if (props.row.tag && props.row.tag.length > 0) {
         tags = props.row.tag.map(tag => {
-            return <span key={tag} className="govuk-tag govuk-!-margin-right-1">{tag}</span>;
+            const tagDetails = fetchTagType([tag]);
+            let tagName = "DEFAULT";
+            let tagLabel= "govuk-tag govuk-!-margin-right-1";
```

Comment:
My gut feeling with these is that this should be returned within the tagType:L4 default object? I also think defaulting the name to the tag value e.g., `REJECTED_BY_PSU` would be more beneficial than `DEFAULT`?

Reply:
My thought process was like if any requirement for adding a new tag comes in future then DEV will add this in hocs-workflow but the corresponding expected data would be missing in hocs-frontend config which would cause frontend to fail instead if we have this configuration UI would display DEFAULT so that DEV would have a chance to fix this by adding it to config.

Let's discuss in today's call.

Reply:
Thanks for your feedback and now code looks nice & tidy.